emscripten.py

```diff
@@ -126,18 +126,13 @@ def update_settings_glue(wasm_file, metadata):
     settings.DEFAULT_LIBRARY_FUNCS_TO_INCLUDE = []
   else:
     syms = settings.DEFAULT_LIBRARY_FUNCS_TO_INCLUDE + metadata.imports
-    syms = set(syms).difference(metadata.exports)
+    syms = set(syms).difference(metadata.function_exports)
```

Comment:
Yes that might be more correct here.

Reply:
Although the only code was only passing functions (and tags) since that is what `.exports` previously contained.. so this could be a semantic change.. but I think for the better.

Reply:
Done.